### DISCUSSION TITLE
Filter usability - choose whether a group should appear in the filters

### DISCUSSION THREAD

**Discussion by jdjoshuadavison** [directus/directus#19707] [Feature Requests]
### The overarching problem
Users find the search and filters difficult to use.
This is one of a number of requests to help solve this problem.

### Field groups
Field groups sometimes make the item view easier to use but make the filters list harder to use. A common cause for this is when groups exist to conditionally show/hide a group of fields - whilst useful in item view, grouping these fields in the filters list usually adds undesired complexity.

My suggestion is to add a “use in filters” option to the group field settings. This will allow admins to choose whether to apply a group in the filters list or not - the fields within the group will still be shown in the filters, they just won’t be grouped.

### Motivation

Users find the filters difficult to use.

### Detailed Design

When editing a group, there will be a toggle to choose whether to use the group in filters or not. It will be enabled by default to maintain a consistent experience for existing directus admins.
When the option is disabled, the fields in the group will be shown in the list of filters without the group.
![use-in-filters](https://github.com/directus/directus/assets/34007858/5594dcd1-096d-4a82-9b54-699efb4a052a)

Metadata group - use in filters *enabled*
![use-in-filters-enabled](https://github.com/directus/directus/assets/34007858/960d2746-d1b0-404a-8a63-0ed7bcdbf769)

Metadata group - use in filters *disabled* (title, short description, long description are visible without a group)
![use-in-filters-disabled](https://github.com/directus/directus/assets/34007858/addfab20-29f5-4eb9-8661-b931429cf0d5)

### Must Have:
- “Use in filters toggle” in the group field settings

### Drawbacks

None

### Alternatives

Groups could simply be expanded by default in the filter list.

### Adoption Strategy

The feature will be set to ‘enabled’ by default. This means if admins don’t change the value from its default, the behaviour will remain consistent with how its always been.

**Comment by jdjoshuadavison** [comment]
Related #19706 
Related #19705

**Comment by luismts** [comment]
### Enhancements and Desired Features for Directus
I have created a thread with a list of features and enhancements that could further drive the utility and versatility of Directus.

You can see it [here](https://gist.github.com/luismts/ed40fb3ce57d0697f604a8ef59a2c875).
